lib/spack/docs/getting_started.rst

```diff
@@ -53,6 +53,13 @@ in the ``SPACK_ROOT`` environment variable.  Add ``$SPACK_ROOT/bin``
 to your path and you're ready to go:
 
 .. code-block:: console
+   
+   # For bash/zsh users
+   $ export SPACK_ROOT=/path/to/spack
+
+   # For tcsh or csh users (note you must set SPACK_ROOT)
+   $ setenv SPACK_ROOT /path/to/spack
+   
 
    $ export PATH=$SPACK_ROOT/bin:$PATH
```

Comment:
This line is bash/zsh specific. We should also add a line for fish now that we support fish.

Reply:
Added in the lines for fish users

Reply:
This line is still bash/zsh specific

Reply:
I've added the line for fish

Reply:
The line for fish has been added, but the `export PATH=` line only works on bash/zsh. We should move this up and add the equivalent logic for other shells. Or we could just add the logic for `PATH` and ignore `SPACK_ROOT`, assuming it is already known to the user. That's what the existing docs were trying to get at.